modules/kml/test/kml-loader.spec.js

```diff
@@ -24,13 +24,13 @@ test('KMLLoader#loader conformance', (t) => {
 test('KMLLoader#testText', async (t) => {
   const response = await fetchFile(KML_URL);
   const KML = await response.text();
-
-  let isKML = KML.startsWith(KMLLoader.tests[0]);
-  t.equal(isKML, true, 'Correctly accepted valid KML');
-
-  isKML = INVALID_KML.startsWith(KMLLoader.tests[0]);
-  t.equal(isKML, false, 'Correctly rejected invalid KML');
-
+  const a = (KMLLoader.tests && KMLLoader.tests[0]) || '';
+  if (typeof a === 'string') {
```

Comment:
Why do you need this `typeof`? For one, when will `KMLLoader.tests[0]` ever not exist? And two, if that doesn't exist, then `a` will be `''`, so this if clause will always run.

Can I suggest also naming the variable better than just `a`? Maybe

```js
const KMLTest = KMLLoader.tests && KMLLoader.tests[0]
if (KMLTest) {
  // check test
}
```

Reply:
> Why do you need this `typeof`? For one, when will `KMLLoader.tests[0]` ever not exist? And two, if that doesn't exist, then `a` will be `''`, so this if clause will always run.

I inserted it here as the variable can be a different type, callback or array buffer, not only string

Reply:
> Can I suggest also naming the variable better than just `a`? Maybe

Changed to 'KMLTest', thanks for your notice :)